src/index.tsx

```diff
@@ -18,7 +18,8 @@ async function init(): Promise<void> {
     const state = store.getState()
     const { user } = state
     const omitRoutes: ReadonlyArray<any> = ['/', '/login']
-    const notNeedOuath = omitRoutes.includes(pathname)
+    const worksNoAuth = [pathname].filter(x => typeof x === 'string' && x.indexOf('/works') > -1)
+    const notNeedOuath = omitRoutes.includes(pathname) || worksNoAuth.length
```

Comment:
This may not be the best way to implement it, but this should allow for users to see works even without logging in.